app/pages/docs/css.mdx

```diff
@@ -81,30 +82,30 @@ export function Button() {
 }
 ```
 
-CSS Modules are an _optional feature_ and are **only enabled for files with the `.module.css` extension**.
-Regular `<link>` stylesheets and global CSS files are still supported.
+CSS Modules is een _optionele feature_ en wordt **enkel ingeschakeld voor bestanden met de `.module.css` extensie**. 
+Gewone `<link>` stylesheets en globale CSS bestanden worden nog steeds ondersteund.
 
-In production, all CSS Module files will be automatically concatenated into **many minified and code-split** `.css` files.
-These `.css` files represent hot execution paths in your application, ensuring the minimal amount of CSS is loaded for your application to paint.
+In productie zullen alle CSS Module-bestanden automatisch worden samengevoegd tot **vele verkleinde en code-split** `.css` bestanden.
+Deze `.css` bestanden vertegenwoordigen hot execution paths in je applicatie, en zorgen ervoor dat de minimale hoeveelheid CSS wordt geladen om je applicatie weer te geven.
 
-## Sass Support {#sass-support}
+## Sass ondersteuning {#sass-support}
 
-Blitz allows you to import Sass using both the `.scss` and `.sass` extensions.
-You can use component-level Sass via CSS Modules and the `.module.scss` or `.module.sass` extension.
+Met Blitz kan je zowel via de `.scss` als `.sass` extensies Sass importeren.
+Je kan Sass op componentniveau gebruiken via CSS-modules en via een van de twee beschikbare extensies.
```

Comment:
Je gebruikt hier **CSS-modules** en bij de rest **CSS Modules**. We moeten er één kiezen. Mijn voorkeur gaat uit naar **CSS-modules**.